### DISCUSSION TITLE
Error with oidc and user groups in latest version 2.5.289

### DISCUSSION THREAD

**Discussion by xrkolovos** [requarks/wiki#5709] [Error / Bug Report]
Hello 
This code https://github.com/requarks/wiki/commit/ebf4da9bea59590eaa6820c6b00b3a1cf1d8c083
has code an issue.

I get the error "cannot read properties of undefined (reading 'map')" after returing from our Auth Server.

**Comment by ghost** [comment]
does this error occur when using the Azure Active Directory Authentication strategy?

**Reply by NGPixel** [reply]
Not related.

**Comment by bwesselb** [comment]
I am seeing this issue as well as soon as I am enabling the `Maps Groups` switch. 
With `Map Groups` disabled I am able to login to the wiki.
